doc/reference/reference_lua/checks.rst

```diff
@@ -0,0 +1,506 @@
+..  checks-module:
+
+Module checks
+=============
+
+**Since:** :doc:`2.11.0 </release/2.11.0>`
```

Comment:
> It is available as a separate rock for all relevant Tarantool versions (1.10+)

Added a note after the introduction: https://docs.d.tarantool.io/en/doc/embedded-checks/reference/reference_lua/checks/.

> Embedded checks package is provided with core Tarantool since 2.11.1

Could you please clarify what do you mean by `Embedded checks package`? I have 2.11.0 on my machine and I can use `checks` without loading it explicitly:

```
tarantool
Tarantool 2.11.0-0-g247a9a4
type 'help' for interactive help
tarantool> checkers.
checkers.datetime(  checkers.error(     checkers.interval(  checkers.uint64(    checkers.uuid_bin(  
checkers.decimal(   checkers.int64(     checkers.tuple(     checkers.uuid(      checkers.uuid_str(  
```